offer-manager-domain/src/main/groovy/life/qbic/business/customers/create/CreateCustomerOutput.groovy

```diff
@@ -11,5 +12,17 @@ import life.qbic.business.UseCaseFailure
  */
 interface CreateCustomerOutput extends UseCaseFailure {
 
+    /**
+     * Is called by the use case, when a new customer has been created
+     * @param message
+     * @deprecated Use the more explicit #customerCreated(Person person) method
+     */
+    @Deprecated
     void customerCreated(String message)
+
+    /**
+     * Is called by the use case, when a new customer resource has been created
+     * @param person The newly created person resource
+     */
+    void customerCreated(Person person)
```

Comment:
Why are you expecting a `Person` here and not a `Customer`. In this case only `Customer`s should be accepted.

Reply:
The UseCase naming is wrong. We do not create customers anymore, but person information entries.

Reply:
:+1: Just noticed that the comment and the method are out of sync